README.md

```diff
@@ -1,13 +1,10 @@
 # Code style for Airlift projects
 
-## IntelliJ IDEA 14 on Mac OS X
-
-To install, copy `IntelliJIdea14/Airlift.xml` into `~/Library/Preferences/IntelliJIdea14/codestyles`
-
-## IntelliJ IDEA 14 on Linux
-
-To install, copy `IntelliJIdea14/Airlift.xml` into `$HOME/.IdeaIC14/config/codestyles`
+## IntelliJ IDEA 2019 or newer
 
+To install, download the `IntelliJIdea2019/Airlift.xml` file.
+In IntelliJ, open "Preferences > Code Style", click the cog icon
```

Comment:
Use bold for UI menu names rather than quotes

Reply:
I'd rather use quotes, why?